dfs/snapshot.go

```diff
@@ -299,7 +299,7 @@ func (dfs *DistributedFilesystem) DeleteSnapshots(tenantID string) error {
 	return nil
 }
 
-func (dfs *DistributedFilesystem) restoreServices(svcs []*service.Service) error {
+func (dfs *DistributedFilesystem) restoreServices(tenantID string, svcs []*service.Service) error {
```

Comment:
Does it make sense to figure out the tenant ID from the list of passed-in services?